.travis.yml

```diff
@@ -23,7 +23,7 @@ install:
   - ./gradlew --refresh-dependencies
   - ./gradlew ciWriteBuildNumber
 script:
-  - travis_wait ./gradlew build -x createExe -S
+  - travis_wait ./gradlew build -S
```

Comment:
@Illyohs The `-x` there means to skip the `createExe` step. 
It seems your commit message is not the same as code change. Which is correct?